### PR TITLE
[AKS] `az aks upgrade`: Add forceupgrade settings to aks stable cli

### PR DESCRIPTION
**Pull request by reneeli123**
**Related command**
add forceupgrade settings to aks stable cli 

**Description**
add forceupgrade settings to aks upgrade which is currently only available in aks update

**Testing Guide**
aks upgrade--resource-group={resource_group} --name={name} --enable-force-upgrade
aks upgrade--resource-group={resource_group} --name={name} --disable-force-upgrade
aks upgrade--resource-group={resource_group} --name={name} --upgrade-override-until 2020-01-01T22:30:17+00:00

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by FumingZhang — COMMENTED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85969392&view=results) to validate the change, test passed!

**Comment by FumingZhang** [comment]
The change is similar to the one in #27258, except that the target command is `az aks upgrade`

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -1827,6 +1827,16 @@
...
     short-summary: Only upgrade node image for agent pools.
+  - name: --enable-force-upgrade
+    type: bool
+    short-summary: Enable forceUpgrade cluster upgrade settings override.
+  - name: --disable-force-upgrade
+    type: bool
+    short-summary: Disable forceUpgrade cluster upgrade settings override.
+  - name: --upgrade-override-until
+    type: string
+    short-summary: Until when the cluster upgradeSettings overrides are effective. It needs to be in a valid date-time format that's within the next 30 days. For example, 2023-04-01T13:00:00Z. Note that if --force-upgrade is set to true and --upgrade-override-until is not set, by default it will be set to 3 days from now.
```
Comment (yanzhudd) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:1838]: It is suggested to mark this help message as "long-summary", and use a brief message as the "short-summary".

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**